ci/ray_ci/tester_container.py

```diff
@@ -113,6 +118,39 @@ def _upload_build_info(self, bazel_log_dir) -> None:
             ]
         )
 
+    def _upload_test_results(self, bazel_log_dir: str) -> None:
+        for test, result in self._get_test_and_results(bazel_log_dir):
+            # TODO(can): upload test results
+            logger.info(
+                f"Updating test object {test.get_name()} with result {result.status}"
```

Comment:
maybe rephrase this log? feels that this line is lying right now.

Reply:
sound good